core/src/test/java/io/questdb/griffin/SecurityTest.java

```diff
@@ -53,6 +54,7 @@ public class SecurityTest extends AbstractGriffinTest {
 
     @BeforeClass
     public static void setUpReadOnlyExecutionContext() {
+        inputRoot = new File(".").getAbsolutePath();
```

Comment:
why are we using current directory in test ?

Reply:
that's to load CSV files. I believe that's a common pattern in all COPY tests - it sets the import root to a current directory and then the COPY command refers to CSV test files as e.g. `copy x from '/src/test/resources/csv/test-import.cs`

Reply:
I might be wrong, but as I remember the pattern is to load resource URI and convert it into path. Current directory is non deterministic

Reply:
that's right. I believe the current directory in a maven multi-module project test depends on how exactly is a test executed. intellij even has (used to have?) an option for this: it could be either the project root or module root. 

I'll check when I can do about it.

Reply:
this pattern is used in all CSV tests:
![image](https://user-images.githubusercontent.com/158619/182186577-4e6870c7-ed2e-44c5-a0da-f3d2ef8a57c0.png)


It has worked well. If you believe we should change it then I'd prefer to do it as a separate PR. as it's not really related to this unification work.

Reply:
lets change it please, sometimes maven runs directly from `core`, others from level above